arch/sparc/core/reset_trap.S

```diff
@@ -12,6 +12,18 @@
 
 GTEXT(__sparc_trap_reset)
 SECTION_FUNC(TEXT, __sparc_trap_reset)
+#ifdef CONFIG_SPARC_SVT
+#ifdef CONFIG_SOC_SPARC_LEON
+	/* On LEON, enable single vector trapping by setting ASR17.SV. */
+	rd	%asr17, %g1
+	set	(1<<13), %g2
+	or	%g1, %g2, %g1
+	wr	%g1, %asr17
+#else
+#error Don't know how to enable SVT on this SOC
```

Comment:
Very minor item. This #error message should probably use quotation marks. The absence of them generates a warning about a missing ' character from the contraction _Don't_ when compiling for that case.

Reply:
Fixed. Thanks!